geode-core/src/integrationTest/java/org/apache/geode/cache30/ShorteningExpirationTimeRegressionTest.java

```diff
@@ -52,68 +46,54 @@
 
   private static final int LONG_WAIT_MS = 2 * 60 * 1000;
   private static final int SHORT_WAIT_MS = 1;
-
   private static final String KEY = "key";
 
-  private String uniqueName;
-  private String regionName;
+  @ClassRule
+  public static ServerStarterRule server = new ServerStarterRule().withAutoStart();
```

Comment:
My main objection to using the ServerStarterRule is that this test does not need a cache server. Cache server is used to allow clients to connect. This test has no clients.
What this test needs is a Cache. It seems reasonable to refactor ServerStarterRule by removing the parts of it that start up the Cache into a CacheStarterRule that would extend MemberStarterRule and ServerStarterRule would be changed to extend CacheStarterRule.

Reply:
yeah, sounds like a good idea. I can add that on to this ticket.